src/Symfony/Component/Asset/UrlPackage.php

```diff
@@ -62,7 +61,7 @@ public function __construct($baseUrls = array(), VersionStrategyInterface $versi
         $sslUrls = $this->getSslUrls($baseUrls);
 
         if ($sslUrls && $baseUrls !== $sslUrls) {
-            $this->sslPackage = new UrlPackage($sslUrls, $versionStrategy);
+            $this->sslPackage = new self($sslUrls, $versionStrategy);
```

Comment:
No, but it's easier when renaming the class name and apart from that, works exactly the same.